src/api/routers/participants/participantsRouter.ts

```diff
@@ -1,675 +1,139 @@
-  setSiteClientTypes,
-  updateApiKeyRoles,
-  updateKeyPair,
-} from '../../services/adminServiceClient';
-import {
-  mapAdminApiKeysToApiKeyDTOs,
-  ParticipantApprovalResponse,
-} from '../../services/adminServiceHelpers';
-import {
-  createdApiKeyToApiKeySecrets,
-  getApiKey,
-  getApiRoles,
-  validateApiRoles,
-} from '../../services/apiKeyService';
-import {
-  constructAuditTrailObject,
-  performAsyncOperationWithAuditTrail,
-} from '../../services/auditTrailService';
-import { assignApiParticipantMemberRole } from '../../services/kcUsersService';
-import {
-  addSharingParticipants,
-  deleteSharingParticipants,
-  getParticipantsApproved,
-  getParticipantsAwaitingApproval,
   ParticipantRequest,
-  sendParticipantApprovedEmail,
   updateParticipant,
-  updateParticipantAndTypesAndApiRoles,
-  UpdateSharingTypes,
   UserParticipantRequest,
 } from '../../services/participantsService';
-import { getSignedParticipants } from '../../services/signedParticipantsService';
-import { getAllUserFromParticipant } from '../../services/usersService';
 import { createBusinessContactsRouter } from '../businessContactsRouter';
 import { createParticipantUsersRouter } from '../participantUsersRouter';
-import { getParticipantAppNames, setParticipantAppNames } from './participantsAppIds';
-import { createParticipantFromRequest, handleCreateParticipant } from './participantsCreation';
-import { getParticipantDomainNames, setParticipantDomainNames } from './participantsDomainNames';
-import { getParticipantKeyPairs } from './participantsKeyPairs';
-import { getParticipantUsers, handleInviteUserToParticipant } from './participantsUsers';
-
-export type AvailableParticipantDTO = Required<Pick<ParticipantDTO, 'name' | 'siteId' | 'types'>>;
+import {
+  handleAddApiKey,
+  handleDeleteApiKey,
+  handleGetParticipantApiKey,
+  handleGetParticipantApiKeys,
+  handleUpdateApiKey,
+} from './participantsApiKeys';
+import { handleGetParticipantAppNames, handleSetParticipantAppNames } from './participantsAppIds';
+import {
+  handleApproveParticipant,
+  handleGetApprovedParticipants,
+  handleGetParticipantsAwaitingApproval,
+} from './participantsApproval';
+import {
+  handleCreateParticipant,
+  handleCreateParticipantFromRequest,
+} from './participantsCreation';
+import {
+  handleGetParticipantDomainNames,
+  handleSetParticipantDomainNames,
+} from './participantsDomainNames';
+import {
+  handleAddKeyPair,
+  handleDeleteKeyPair,
+  handleGetParticipantKeyPairs,
+  handleUpdateKeyPair,
+} from './participantsKeyPairs';
+import {
+  handleAddSharingPermission,
+  handleGetSharingPermission,
+  handleRemoveSharingPermission,
+  handleUpdateSharingTypes,
+} from './participantsSharingPermissions';
+import { handleGetSignedParticipants } from './participantsSigned';
+import { handleGetParticipantUsers, handleInviteUserToParticipant } from './participantsUsers';
+
+const handleUpdateParticipant = async (req: UserParticipantRequest, res: Response) => {
```

Comment:
A few of these handlers still made sense to me to belong in this file, mainly because I couldn't think of a good file name that would indicate that they are directly related to the participant. Open to suggestions though.